### PR TITLE
REPL and Tips for Methods

### LINKED ISSUE #92: Update methods for REPL/Tips
Update C# Getting Started:

- [x] Examples use REPL format
- [x] Add tips for newLanguage

Syntax:

### PR DESCRIPTION
**Pull request by ardalis**
Fixes #92

### CODE REVIEW

File: content/csharp/getting-started/methods.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ static int Add(int operand1, int operand2)
 string FullName()
 {
     // FirstName and LastName would be Properties of the class
-    // this method appears on
+    // this method appears on - shouldn't be static
```
Comment (efleming18) [inline content/csharp/getting-started/methods.md]: I guess this comment doesn't make much sense to me?

**Comment by efleming18** [comment]
:watch:

File: content/csharp/getting-started/methods.md (hunk 1 above)
Comment (benrick) [inline content/csharp/getting-started/methods.md]: It's one comment, not two. I think your confusion is based on the break.

**Comment by benrick** [comment]
:shipit: @ardalis

**Pull request merged**